src/unix/nto/mod.rs

```diff
@@ -641,7 +668,9 @@ s_no_extra_traits! {
 
     pub struct sigevent {
```

Comment:
Why was the extra padding added?  The structure itself is 32 bytes on 64 arches on QNX.  With these changes, the structure is larger than our internal implementation.  For a safety focused language, this seems like a strange thing to do.

Reply:
The size should not have changed. The second member `__sigev_un1` with type usize (8 bytes with 8-byte-alignment) does not actually exist on `C` side but is a union. I have removed `__sigev_un1`.

The union contains an `int` (4 bytes) as signal number and other members of pointer type (64 bit aligned on 64 bit machines). If I'm not mistaken, all members of the union start at the same address. It is the address of the member with the highest alignment requirements, i.e. the signal number gets an 8-byte-alignment (even though 4 bytes would be sufficient for an int). The `padding1` member ensures this alignment.
The alignment and struct size is also confirmed by the ctest2 check.

What is your suggestion to make the signal number accessible?

Reply:
```text
    //         |4b         |4b         |4b         |4b       |4b       |
    // before: |notify     | padding(*)|    un1 (usize)      |value    |
    // after : |notify     | pad1      | signo     | pad2    |value    |
```
(*) Caused by alignment requirement of next member un1 (usize)